boards/wch/linkw/doc/index.rst

```diff
@@ -0,0 +1,116 @@
+.. zephyr:board:: linkw
+
+Overview
+********
+
+The `WCH`_ LinkW hardware provides support for QingKe 32-bit RISC-V4C
+processor and the following devices:
+
+* CLOCK
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+
+The board is equipped with two LEDs and two Buttons.
+The `WCH webpage on CH32V208`_ contains the processor's manuals.
+The `WCH webpage on LinkW`_ contains the LinkW's schematic.
+
+Hardware
+********
+
+The QingKe 32-bit RISC-V4C processor of the WCH LinkW is clocked by an external
+32 MHz crystal or the internal 8 MHz oscillator and runs up to 144 MHz.
+The CH32V208 SoC Features 4 USART, 4 GPIO ports, 2 SPI, 2 I2C, ADC, RTC,
+CAN, 2 USB Device, USB Host, OPA, ETH with PHY, several timers, and BLE 5.3.
+
+Supported Features
+==================
+
+The ``linkw`` board target supports the following hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| PINCTRL   | on-chip    | pinctrl              |
++-----------+------------+----------------------+
+| TIMER     | on-chip    | timer                |
++-----------+------------+----------------------+
+| UART      | on-chip    | uart                 |
++-----------+------------+----------------------+
+
+Other hardware features have not been enabled yet for this board.
+
+Connections and IOs
+===================
+
+LED
+---
+
+* LED0 = Green Mode LED
+* LED1 = Blue Activity LED
+
+Button
+------
```

Comment:
Missing newline... if that matters at all!

Reply:
Done.